util/admin/admin.go

```diff
@@ -440,7 +440,7 @@ func CheckRecordAndIndex(sessCtx sessionctx.Context, txn kv.Transaction, t table
 		for i, val := range vals1 {
 			col := cols[i]
 			if val.IsNull() {
-				if mysql.HasNotNullFlag(col.Flag) {
+				if mysql.HasNotNullFlag(col.Flag) && col.ToInfo().OriginDefaultValue == nil {
 					return false, errors.New("Miss")
```

Comment:
It is better to add more detail error information and make diagnose easier.

Reply:
Right, we should create a better error message.

Reply:
done. PTAL